specs/ExtendedProcessFailed.md

```diff
@@ -0,0 +1,671 @@
+  COREWEBVIEW2_PROCESS_FAILED_KIND_RENDER_PROCESS_UNRESPONSIVE,
+
+  // New values.
+  
+  /// Indicates that a frame-only render process ended unexpectedly. The process
+  /// exit does not impact the top-level document, only a subset of the
+  /// subframes within it. The content in these frames is replaced with an error
+  /// page in the frame.
+  COREWEBVIEW2_PROCESS_FAILED_KIND_FRAME_RENDER_PROCESS_EXITED,
+
+  /// Indicates that a utility process ended unexpectedly.
+  COREWEBVIEW2_PROCESS_FAILED_KIND_UTILITY_PROCESS_EXITED,
+
+  /// Indicates that a sandbox helper process ended unexpectedly.
+  COREWEBVIEW2_PROCESS_FAILED_KIND_SANDBOX_HELPER_PROCESS_EXITED,
+
+  /// Indicates that the GPU process ended unexpectedly.
+  COREWEBVIEW2_PROCESS_FAILED_KIND_GPU_PROCESS_EXITED,
+
+  /// Indicates that a PPAPI plugin process ended unexpectedly.
+  COREWEBVIEW2_PROCESS_FAILED_KIND_PPAPI_PLUGIN_PROCESS_EXITED,
+
+  /// Indicates that a PPAPI plugin broker process ended unexpectedly.
+  COREWEBVIEW2_PROCESS_FAILED_KIND_PPAPI_BROKER_PROCESS_EXITED,
+
+  /// Indicates that a process of unspecified kind ended unexpectedly.
+  COREWEBVIEW2_PROCESS_FAILED_KIND_UNKNOWN_PROCESS_EXITED,
+} COREWEBVIEW2_PROCESS_FAILED_KIND;
+
+
+/// Specifies the process failure reason used in the
+/// `ICoreWebView2ProcessFailedEventHandler` interface.
+[v1_enum]
+typedef enum COREWEBVIEW2_PROCESS_FAILED_REASON {
+  /// An unexpected process failure occurred.
+  COREWEBVIEW2_PROCESS_FAILED_REASON_UNEXPECTED,
+
+  /// The process became unresponsive.
+  /// This only applies to the main frame's render process.
+  COREWEBVIEW2_PROCESS_FAILED_REASON_UNRESPONSIVE,
+
+  /// The process was killed. For example, from Task Manager.
+  COREWEBVIEW2_PROCESS_FAILED_REASON_KILLED,
+
+  /// The process crashed.
+  COREWEBVIEW2_PROCESS_FAILED_REASON_CRASHED,
+
+  /// The process failed to launch.
+  COREWEBVIEW2_PROCESS_FAILED_REASON_LAUNCH_FAILED,
+
+  /// The process died due to running out of memory.
+  COREWEBVIEW2_PROCESS_FAILED_REASON_OUT_OF_MEMORY,
+} COREWEBVIEW2_PROCESS_FAILED_REASON;
+
+/// A continuation of `ICoreWebView2ProcessFailedEventArgs` interface.
+[uuid(c62f5687-2f09-481b-b6e4-2c0620bc95a2), object, pointer_default(unique)]
+interface ICoreWebView2ProcessFailedEventArgs2 : IUnknown {
+  // The ProcessFailedKind property below is already in
+  // ICoreWebView2ProcessFailedEventArgs. The changes to this interface extend
+  // its return enum so it can now specify additional process kinds: gpu process
+  // exited, utility process exited, etc. The property is included here to show
+  // the updated docs.
+
+  /// The kind of process failure that has occurred. `processFailedKind` is
+  /// `COREWEBVIEW2_PROCESS_FAILED_KIND_RENDER_PROCESS_EXITED` if the
+  /// failed process is the main frame's renderer, even if there were subframes
+  /// rendered by such process; all frames are gone when this happens.
+  // [propget] HRESULT ProcessFailedKind(
```

Comment:
Property name of `Kind` should be sufficient, but it looks like it's too late to simplfy -- this already shipped in v1.